modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -581,6 +584,42 @@ private JSONArray _getSelectedItemsJSONArray(
 		return jsonArray;
 	}
 
+	private JSONArray _getSortsJSONArray(
+		ObjectDefinition fdsViewObjectDefinition, ObjectEntry fdsViewObjectEntry) {
+
+		try {
+			List<ObjectEntry> fdsSortingObjectEntries = new ArrayList<>(
+				_getRelatedObjectEntries(
+					fdsViewObjectDefinition, fdsViewObjectEntry, "fdsViewFDSSortRelationship"));
+
+			if (ListUtil.isEmpty(fdsSortingObjectEntries)) {
+				return _jsonFactory.createJSONArray();
+			}
+
+			return JSONUtil.toJSONArray(
+				fdsSortingObjectEntries,
+				(ObjectEntry fdsSortingObjectEntry) -> _getSortsJSONObject(
+					fdsSortingObjectEntry));
+		}
+		catch (Exception exception) {
+			if (_log.isWarnEnabled()) {
+				_log.warn(
+					"Unable to create frontend data set sort from frontend data set view", exception);
+			}
+			return _jsonFactory.createJSONArray();
+		}
```

Comment:
iirc I had to add this time ago to avoid the situation of sending a null/undefined value. Correct behavior should be not sending `sorts` if there are not. And if there are, an array should be sent. An error generating the array should probably be equivalent to send nothing about sorts